museum/sampleRecords/adminDonor.destination.json

```diff
@@ -0,0 +1,1025 @@
+{
+  "ok":true,
+  "results":[
+    {
+      "object_id":"1",
+      "id":"1",
+      "idno":"A1988.135",
+      "display_label":"STRAWBERRY HILL FARM, ALBANY",
+      "ca_objects.AccessionPrefix":"A",
+      "ca_objects.AccessionYear":"1988",
+      "ca_objects.AccessionNumber":"135",
+      "ca_objects.AccessionSeries":"0",
+      "ca_objects.AccessionBy":"jm",
+      "ca_entities":"UNKNOWN",
+      "ca_entities.entity_id":"1",
+      "ca_entities.preferred_labels.surname":"UNKNOWN",
+      "ca_entities.Phone":";;;;;;",
+      "ca_entities.MobilePhone":";;;;;;",
+      "ca_entities.Fax":";;;;;;",
+      "ca_entities.Email":";;;;;;",
+      "ca_entities.DonorNotes":";;;;;;",
+      "ca_entities.Deceased":";;;;;;",
+      "ca_entities.DonorAgentName":";;;;;;",
+      "ca_entities.DonorAgentAddress":";;;;;;",
+      "ca_entities.DonorAgentPhone":";;;;;;",
+      "ca_entities.LastEditBy":"urbaj",
+      "ca_entities.LastEditDate":"March 6 2015",
```

Comment:
Strange date format, should be ISO format, `YYYY-MM-DD` or AU format, `DD MMM YYYY`.